ai-engineering/measure.mdx

```diff
@@ -301,6 +245,14 @@ To run your evaluation suites from your terminal, [install the Axiom CLI](/refer
 | Run eval by name | `axiom eval "spam-classification"` |
 | List available evals without running | `axiom eval --list` |
 
+## Custom scorers
+
+A scorer is a function that scores a capability’s output. Scorers receive the `input`, the generated `output`, and the `expected` value, and return a score.
+
+The `autoevals` library that provides prebuilt scorers for common tasks like semantic similarity, factual correctness, and text matching
```

Comment:
This isn't a full sentence I think? Maybe something like

There are many libraries with prebuilt scorers for common tasks like semantic similarity and text matching that you can wrap, such as `autoevals`.